### ISSUE TITLE
Make Redis Object Cache Full compatible with the Batcache Plugin

### ISSUE THREAD

**Issue by portablesoft** [tillkruss/Redis-Object-Cache#17]
Hi,

Batcache is a full page cache plugin using an object cache plugin. It could cache pages in the memory.

I install the Redis Object Cache and Batcache. However, I could not get the cached page even I refresh the page for many times. Batcache just recreate a new page cache when I refresh the page. It did not send the old cached page.

Is there any compatible issues between Redis Object Cache and the Batcache Plugin? Thanks.

Batcache: https://wordpress.org/plugins/batcache/

**Comment by jeremyclarke** [comment]
Are you sure that the Redis object cache is working in general? It could be that Batcache is failing because you don't have this plugin configured properly. 

You can test by enabling the Debug Bar plugin and reviewing the Object Cache section, which should list all the internal variables being stored in Redis and cached across pages. Specifically you should notice that disabling the Redis Object Cache causes the number of "misses" to go way up compared to when it's active.

**Comment by tillkruss** [comment]
@rohmann submitted 2 PRs for Batcache compatibly. It should work. See #9 and #11.

**Comment by portablesoft** [comment]
@jeremyclarke @tillkruss thanks. I use the advanced-cache.php by @rohmann and the cache work well now.

However, the advanced-cache.php from the official WordPress plugin site could still not work correctly. I noticed that the plugin was 'Last Updated: 2 weeks ago'.

So I think the new version of Batcache could not use Redis Object Cache as backend now. But I'm only a WordPress user not a developer, I could not find what was changed and why it fails on Redis Object Cache.

@rohmann 's Batcache: https://github.com/rohmann/batcache
Batcache on the WordPress plugin: https://wordpress.org/plugins/batcache/
